R/get_pids.R

```diff
@@ -32,7 +32,6 @@ get_pids <- function(pkg_doi, data_url = NULL){
     pkg_doi <- stringr::str_replace(pkg_doi, "https://", "")
     
     # check data_url and decode; try to grab pid
-    stopifnot(is.character(data_url))
```

Comment:
Thanks for the PR!  I think I'll handle this issue a little differently, so I'll close this PR and make the changes directly.

Rather than getting rid of the stopifnot, I'll move it inside the if(!is.null) expression